### PR TITLE
More doc tweaks

### PR DESCRIPTION
**Pull request by cijothomas**
Fixes #.

## Changes

Please provide a brief description of the changes here.

For significant contributions please make sure you have completed the following items:

* [ ] Appropriate `CHANGELOG.md` updated for non-trivial changes
* [ ] Design discussion issue #
* [ ] Changes in public API reviewed

### CODE REVIEW

File: docs/trace/customizing-the-sdk/README.md [hunk 1]
```diff
@@ -252,12 +268,19 @@ The snippet below shows configuring a custom sampler to the provider.
...
+to learn about how to write own samplers.
+
 ## Context Propagation
 
 // TODO: OpenTelemetry Sdk contents about Context. // TODO: Links to built-in
 instrumentations doing Propagation.
+
+## Configuration using Dependency Injection
+
+// TODO: Placeholder
```
Comment (cijothomas) [inline docs/trace/customizing-the-sdk/README.md:286]: @CodeBlanch This is all yours!

File: docs/trace/customizing-the-sdk/README.md [hunk 2]
```diff
@@ -142,7 +146,7 @@ it is built.
 using OpenTelemetry;
 using OpenTelemetry.Trace;
 
-using var tracerProvider = Sdk.CreateTracerProviderBuilder()
+var tracerProvider = Sdk.CreateTracerProviderBuilder()
```
Comment (utpilla) [inline docs/trace/customizing-the-sdk/README.md:149]: Add `tracerProvider.Dispose` for these examples as well?

**Review by vishweshbankwar — APPROVED** [review]

**Review by utpilla — APPROVED** [review]

File: docs/trace/customizing-the-sdk/README.md (hunk 2 above)
Comment (cijothomas) [reply docs/trace/customizing-the-sdk/README.md:149]: do we need? they are not full code, but just snippets.

Comment (utpilla) [reply docs/trace/customizing-the-sdk/README.md:149]: > they are not full code, but just snippets.

Why remove `using` in the first place? 😄

I guess if we are removing `using` then we could add these three lines to snippets to stay consistent and encourage best practices.
```csharp
// ....
// Dispose at application shutdown
tracerProvider.Dispose()
```

Comment (cijothomas) [reply docs/trace/customizing-the-sdk/README.md:149]: > Why remove using in the first place? 😄

Have seen a lot of users copying it as-is, and unintentionally disposing too early!.

**Pull request merged**